crates/shared/src/price_estimation/competition.rs

```diff
@@ -55,18 +55,17 @@ impl PriceEstimating for CompetitionPriceEstimator {
 }
 
 #[derive(Debug)]
-struct EstimateData<'a> {
-    estimator_name: &'a str,
+struct EstimateData {
```

Comment:
Note that this change makes it so the `winning_estimate` log above no longer prints the estimator that was used.